deployment/lib/cf-url-rewriter.js

```diff
@@ -0,0 +1,59 @@
+const aws = require('aws-sdk');
+
+const s3 = new aws.S3({ apiVersion: '2006-03-01' });
```

Comment:
That a pretty old version, can we omit the version keyword and get the latest?

Reply:
The current code is mostly referring to SDK V2. Let me take a look at the SDK V3 https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html

Reply:
the latest S3 API version is `2006-03-01` https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html

Reply:
No longer applicable after moving to read `index.json`